test/include/vikunja/test/AlpakaSetup.hpp

```diff
@@ -50,6 +50,62 @@ namespace vikunja
                 , queueAcc{devAcc}
             {
             }
+
+            /**
+             * @brief Allocate 1D memory on host.
```

Comment:
```suggestion
             * @brief Allocate 1D memory on the host.
```

Also in the other places below.